speech/README.md

```diff
@@ -22,9 +22,10 @@ Usage:
     Usage: ruby speech_samples.rb <command> [arguments]
 
     Commands:
-      recognize               <audio-file.raw>
-      async_recognize         <audio-file.raw>
-      async_recognize_results <operation name>
+      recognize           audio-file.raw
```

Comment:
This is more of an example than a description this command.

This comment applies to the other commands:

* recognize_gcs
* async_recognize
* async_recognize_gcs

Referencing: https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/master/storage/files.rb#L317